### DISCUSSION TITLE
DB Migration from Elsa 1.x to Elsa 2.0

### DISCUSSION THREAD

**Discussion by Aoshua** [elsa-workflows/elsa-core#970] [Q&A] [answered]
Were there any changes to the database schema of Elsa 2.0 such that moving from 1.x to 2.0 will require running a DB migration?

**Comment by sfmskywalker** [comment, marked as answer]
Yes, in fact there have been many breaking changes. Basically workflows will have to be recreated and migrating workflow instances might not be worth the risk of making mistakes. If you have many workflow definitions and or instances, the better strategy might be to launch Elsa 2 separately until after existing Elsa 1 workflow instances complete and new runs start in Elsa 2.
